operations/helm/charts/mimir-distributed/templates/alertmanager/alertmanager-dep.yaml

```diff
@@ -46,6 +46,9 @@ spec:
             - "-target=alertmanager"
             - "-config.expand-env=true"
             - "-config.file=/etc/mimir/mimir.yaml"
+          {{ if .Values.alertmanager.fallbackConfig }}
+            - "--alertmanager.configs.fallback=/etc/alertmanager/fallback-config.yaml"
```

Comment:
Can you please move this to the `mimir.config` blob in values.yaml? This is where we keep most of the config. The flag you are setting doesn't apply to all components, but that's ok. It will also not break users that have already set a fallback config in their `mimr.unstructuredConfig` (because CLI flags take precedence.

Reply:
moved to mimir.config.alertmanager.fallback_config_file